code/__DEFINES/access.dm

```diff
@@ -33,7 +33,7 @@
 #define ACCESS_BRIGPHYS 34
 #define ACCESS_HYDROPONICS 35
 #define ACCESS_LIBRARY 37
-#define ACCESS_LAWYER 38
+#define ACCESS_IAA 38
```

Comment:
Is the rebranding needed? They are basically the lawyer+. 
Also we already have IAAs so the name is conflicting.

Reply:
The rebranding is needed because they aren't prisoner defense lawyers, they're agents sent by NT to make sure the station runs smoothly and the station is in line with NT's interests.

Also I renamed IAAs (the job) to CAAs (Corporate Affairs Agents)